src/prisma/schema.prisma

```diff
@@ -1367,8 +1366,8 @@ model DraftProposalRubricCriteriaAnswer {
   userId           String
   response         Json
   comment          String?
-  evaluation       ProposalEvaluation?    @relation(fields: [evaluationId], references: [id])
-  evaluationId     String?                @db.Uuid
+  evaluation       ProposalEvaluation     @relation(fields: [evaluationId], references: [id])
```

Comment:
there's other places we can make evaluationId required, but this table does not have any null values at the moment